### DISCUSSION TITLE
Issues Loading Dataset

### DISCUSSION THREAD

**Discussion by rachyuthan** [tudelft/AE4353-Y24#23] [Ex_2] [answered]
I am having some issues loading the dataset at the beginning. 

![image](https://github.com/user-attachments/assets/6f4981b6-9c72-45f7-831c-1be9421719e5)

I am not understanding what the issue is because I copied the path to the directory after using pwd in the directory where the required dataset is.

My dataset_base.yaml file looks like this:
![image](https://github.com/user-attachments/assets/18f8dfe7-10cb-4e65-b0cc-ad73d26d8638)

Any help in solving this issue would be much appreciated

**Comment by yilun-wu** [comment]
You should not need to change the `dataset_path` of `dataset_base.yaml`.

**Reply by yilun-wu** [reply, marked as answer]
If you have downloaded the files to the mounted dataset folder according the `devcontainer.json`, everything should work out-of-the-box.

**Reply by rachyuthan** [reply]
Ah ok I had an issue with how I added the files to the dataset folder, but now it work thanks!
